Source/Core/DolphinWX/TASInputDlg.h

```diff
@@ -33,10 +39,14 @@ class TASInputDlg : public wxDialog
   void CreateWiiLayout(int num);
 
 private:
-  static constexpr int ID_C_STICK = 1001;
-  static constexpr int ID_MAIN_STICK = 1002;
-  static constexpr int ID_CC_L_STICK = 1003;
-  static constexpr int ID_CC_R_STICK = 1004;
+  enum : int
+  {
+    ID_C_STICK = 1001,
+    ID_MAIN_STICK = 1002,
+    ID_CC_L_STICK = 1003,
+    ID_CC_R_STICK = 1004
+  };
+
   int m_eleID = 1005;
```

Comment:
Do you think this should be documented? Just a brief comment should be enough, to make it immediately obvious that it's linked to the enum.